tests/functional_test_base.go

```diff
@@ -119,6 +119,11 @@ func (s *FunctionalTestBase) setupSuite(defaultClusterConfigFile string, options
 
 	clusterConfig, err := GetTestClusterConfig(defaultClusterConfigFile)
 	s.Require().NoError(err)
+	if s.dynamicConfigOverrides == nil {
+		s.dynamicConfigOverrides = make(map[dynamicconfig.Key]interface{})
+	}
+	s.dynamicConfigOverrides[dynamicconfig.HistoryScannerEnabled] = false
+	s.dynamicConfigOverrides[dynamicconfig.TaskQueueScannerEnabled] = false
```

Comment:
why just those two? not executions scanner or build id scavenger? I see those are default off right now but may change to true in the future

Reply:
Makes sense to me, fixed